pcdsutils/requirements.py

```diff
@@ -0,0 +1,207 @@
+import argparse
+import logging
+import pathlib
+import re
+import yaml
+
+
+logger = logging.getLogger(__name__)
+
+
+PIP_REQUIREMENT_FILES = {
+    'requirements.txt': ('host', 'build', 'run'),
+    'dev-requirements.txt': ('test', ),
+    # 'docs-requirements.txt': 'build',  # ?
+}
+
+RE_DEPENDENCY_NAME = re.compile(r'^([a-z0-9_-]+)\s*([><=]?.*)$', re.IGNORECASE)
+CONDA_NAME_TO_PYPI_NAME = {'pyqt': 'pyqt5'}
```

Comment:
I still get a little sad every time I'm reminded about this

Reply:
It is a shame to see this closed without a solution. https://github.com/ContinuumIO/anaconda-issues/issues/1554